functions/src/modules/SQR.functions.ts

```diff
@@ -8,6 +8,20 @@ const lodash = require('lodash');
 
 const db = admin.database();
 import * as helpers from './../helpers';
+import GoogleSheet from '../services/GoogleSheet';
+import {
+  createUpdateLink,
+  spreadsheetDateFormat,
+  withDefault,
+  commaSeparated,
+  EMPTY_VALUE,
+} from '../utils/parsers';
+
+export enum ISoundQualityReportSheet {
+  Allotments = 'Allotments',
+  Submissions = 'Submissions',
+  Statistics = 'Statistics',
```

Comment:
I think we don't need Statistics sheet at all.

Reply:
Removed "Statistics" sheet